packages/components/src/ComboBox/docs/ComboBoxOption/selectionIndicator.tsx

```diff
@@ -9,24 +9,23 @@ export default function Example() {
             aria-label="list of options"
             selectedKey={selectedKey}
             onSelectionChange={setSelectedKey}
+            label="Roles"
+            listBoxProps={{
```

Comment:
selectionIndicator should be directly on the combobox no?

Reply:
yes, it could be. Honestly I wasn't completely sure which props should be moved and which should stay.

Reply:
imagine listbox props being a last resort type of things. We should avoid it for "normal" use cases such as this one

Reply:
like listboxProps to pass a custom border style, or add data-randomProp because you need it for something custom